app/views/shared/data/all/_metadata.json.jbuilder

```diff
@@ -2,7 +2,7 @@ json.object_tag object_tag(object)
 json.object_label label_for(object)
 json.global_id object.persisted? ? object.to_global_id.to_s : nil
 json.base_class object.class.base_class.name
-json.url url_for(only_path: false, format: :json)
+#json.url url_for(only_path: false, format: :json)
```

Comment:
@mjy I noticed that for /taxon_names.json it always shows the same URL (e.g. https://sandcastle.taxonworks.org/taxon_names.json). Here it is commented but in development it is not. Is it used somewhere? Could json.object_url be changed to show the full URL instead? (which would eventually vary if you call from API or internally).

Reply:
I'm pretty sure it is used, likely in an annotator.  It should dynamically generate the whole URL I think.  It might omit the base in development, is that what you are asking?

Reply:
@mjy this is what I mean is happening (look at url and object_url):
```JSON
{
    "id": 44517,
    "name": "rhodiensis",
    "parent_id": 44487,
    "cached": "Saga rhodiensis",
    "cached_html": "\u003ci\u003eSaga rhodiensis\u003c/i\u003e",
    "feminine_name": null,
    "masculine_name": null,
    "neuter_name": null,
    "cached_author_year": "Salfi, 1929",
    "etymology": null,
    "year_of_publication": null,
    "verbatim_author": null,
    "rank": "species",
    "rank_string": "NomenclaturalRank::Iczn::SpeciesGroup::Species",
    "type": "Protonym",
    "created_by_id": 5,
    "updated_by_id": 24,
    "project_id": 2,
    "cached_valid_taxon_name_id": 44517,
    "cached_original_combination": "Saga rhodiensis",
    "cached_original_combination_html": "\u003ci\u003eSaga rhodiensis\u003c/i\u003e",
    "cached_secondary_homonym": "Saga rhodiensis",
    "cached_primary_homonym": "Saga rhodiensis",
    "created_at": "2013-02-12T18:23:00.000Z",
    "updated_at": "2019-03-12T23:47:20.361Z",
    "nomenclatural_code": "iczn",
    "verbatim_name": null,
    "object_tag": "\u003ci\u003eSaga rhodiensis\u003c/i\u003e",
    "object_label": "Saga rhodiensis Salfi, 1929",
    "global_id": "gid://taxon-works/Protonym/44517",
    "base_class": "TaxonName",
    "url": "https://sandcastle.taxonworks.org/taxon_names.json",
    "object_url": "/taxon_names/44517",
    (OUTPUT OMITTED)
}
```

What I'm suggesting is that "object_url" be https://sandcastle.taxonworks.org/taxon_names/44517 and remove "url". However, I don't know if this won't break anything. In any case, taxon-name-api branch has "url" commented out, so it only outputs "object_url". Do I just revert the change?

Reply:
(The JSON code comes from the non-API https://sandcastle.taxonworks.org/taxon_names.json?per=5 using Orthoptera project)

Reply:
Revert for now.